tests/resume.test.ts

```diff
@@ -0,0 +1,56 @@
+import { ActivityScope, ActivityType, SubmitAttendanceForUsersRequest, UserAccessType } from '../types';
+import { ControllerFactory } from './controllers';
+import { DatabaseConnection, EventFactory, PortalState, UserFactory } from './data';
+
```

Comment:
Might be worth handling other common error cases? (request a user resume that isn't uploaded yet, what happens with reuploads)

Reply:
good idea, will look into that